thrift/lib/go/thrift/http_client.go

```diff
@@ -213,8 +213,8 @@ func (p *HTTPClient) Flush() error {
 	if err != nil {
 		return NewTransportExceptionFromError(err)
 	}
-	p.header.Add("Content-Type", "application/x-thrift")
-	req.Header = p.header
+	req.Header = p.header.Clone()
```

Comment:
Any special reason of cloning the `p.header`?

Reply:
Discussed offline. This is to make all `p.header` unchanged since requests will reuse this.